receiver/prometheusexecreceiver/subprocessmanager/manager.go

```diff
@@ -30,14 +31,19 @@ import (
 
 // Run will start the process and keep track of running time
 func (proc *SubprocessConfig) Run(ctx context.Context, logger *zap.Logger) (time.Duration, error) {
-	// Parse the command line string into arguments
-	args, err := shellquote.Split(proc.Command)
-	if err != nil {
-		return 0, fmt.Errorf("could not parse command, error: %w", err)
+	if runtime.GOOS == "windows" {
```

Comment:
Can I suggest the following:

```golang
var childProcess *exec.Cmd
switch runtime.GOOS {
case "windows":
   childProcess = exec.Command(proc.Command)
default:
    args, err = shellqoute.Split(proc.Command)
    if err != nil {
         return 0, err
    }
    childProcess = exec.Command(args[0], args[1:]...)
}
```

If in future there needs to be a seperate way for darwin based systems, it would be easy to add in.

Reply:
Good idea, will do that!